client/my-sites/plans/main.jsx

```diff
@@ -194,8 +195,8 @@ class Plans extends Component {
 				<QueryPlans />
 				<TrackComponentView eventName="calypso_plans_view" />
 				<Main
-					fullWidthLayout={ is2023OnboardingPricingGrid }
-					wideLayout={ ! is2023OnboardingPricingGrid }
+					fullWidthLayout={ is2023OnboardingPricingGrid && ! isEcommerceTrial }
+					wideLayout={ ! ( is2023OnboardingPricingGrid && ! isEcommerceTrial ) }
```

Comment:
Maybe use a positive check here? As written, this is not obvious to me.
```suggestion
					wideLayout={ isEcommerceTrial || ! is2023OnboardingPricingGrid }
```